.github/workflows/build.yml

```diff
@@ -339,7 +340,7 @@ jobs:
         run: ./scripts/safari-build.ps1 -copyonly
 
       - name: Build application (dev)
-        if: github.ref != 'refs/heads/master' || github.ref == 'refs/heads/rc'
```

Comment:
This should actually be `!=` (this is a part of that logic thing that I mentioned needs to be updated)

Reply:
either that, or we can break out the logic of `npm run dist:mac:masdev` and remove that if statement altogether

Reply:
Updated to have both be `!=`.